src/content.js

```diff
@@ -514,6 +523,8 @@ async function onDomReady() {
 
 	markUnread.setup();
 
+	addMarketplaceToProfileDropDown();
+
```

Comment:
You're already using `elementReady`, so you can move this out of `onDomReady` and in `init`